controlplane/eks/api/v1beta2/awsmanagedcontrolplane_webhook.go

```diff
@@ -572,6 +573,13 @@ func (*awsManagedControlPlaneWebhook) Default(_ context.Context, obj runtime.Obj
 	infrav1.SetDefaults_NetworkSpec(&r.Spec.NetworkSpec)
 
 	// Set default value for BootstrapSelfManagedAddons
-	r.Spec.BootstrapSelfManagedAddons = true
+	if r.Spec.BootstrapSelfManagedAddons == nil {
+		r.Spec.BootstrapSelfManagedAddons = aws.Bool(true)
+	}
+
+	// Set default value for AutoMode
+	if r.Spec.AutoMode == nil {
+		r.Spec.AutoMode = aws.Bool(false)
+	}
```

Comment:
shouldn't the default be true considering this was always on by default in previous versions? a change like this could be breaking for users